rosbag2_transport/src/rosbag2_transport/player.cpp

```diff
@@ -1324,4 +1457,146 @@ size_t Player::get_number_of_registered_on_play_msg_post_callbacks()
   return pimpl_->get_number_of_registered_on_play_msg_post_callbacks();
 }
 
+PlayerImpl::PlayerClient::PlayerClient(
+  std::shared_ptr<rclcpp::GenericClient> cli,
+  std::string service_name,
+  std::string service_event_type,
+  const rclcpp::Logger & logger)
+: client_(std::move(cli)),
+  service_name_(service_name),
+  logger_(logger)
+{
+  ts_lib_ = rclcpp::get_typesupport_library(
+    service_event_type, "rosidl_typesupport_cpp");
+
+  ts_ = rclcpp::get_typesupport_handle(
+    service_event_type,
+    "rosidl_typesupport_cpp",
+    *ts_lib_);
+
+  auto ts_handle = get_message_typesupport_handle(
```

Comment:
```suggestion
  auto message_ts_handle = get_message_typesupport_handle(
```
There is some sort of confusion between `ts_` and `ts_handle` - both of them typesupport handles